doc/source/formats/reference/imd.rst

```diff
@@ -0,0 +1,330 @@
+
+MD Engine Configuration
+-----------------------
+
+We provide below example configurations for enabling IMDv3 streaming in popular MD engines.
+
+**GROMACS**
+
+Add IMD settings to your ``.mdp`` file:
+
+.. code-block:: text
+
+    ; IMD settings
+    IMD-group        = System
+    IMD-version      = 3
+    IMD-nst          = 1
+    IMD-time         = No
+    IMD-coords       = Yes
+    IMD-vels         = No
+    IMD-forces       = No
+    IMD-box          = No
+    IMD-unwrap       = No
+    IMD-energies     = No
+
+
+
+Run with IMD enabled:
+
+.. code-block:: bash
+
+    gmx mdrun -v -nt 4 -imdwait -imdport 8889
+
+**LAMMPS**
+
+Use the IMD fix in your input script:
+
+.. code-block:: text
+
+    # IMD setup
+    fix ID group-ID imd <port> trate <frequency> version 3 unwrap <on/off> time <on/off> box <on/off> coordinates <on/off> velocities <on/off> forces <on/off>
+
+Run your LAMMPS simulation as usual.
+
+**NAMD**
+
+Add IMD configuration to your NAMD configuration file:
+
+.. code-block:: text
+
+    # IMD Settings
+    IMDon yes
+    IMDport <port, must be the same port used for the client>
+    IMDwait <yes/no>
+    IMDfreq <frequency of sending data to the client>
+
+    IMDsendPositions <yes/no>
+    IMDsendEnergies <yes/no>
+    IMDsendTime <yes/no>
+    IMDsendBoxDimensions <yes/no>
+    IMDsendVelocities <yes/no>
+    IMDsendForces <yes/no>
+    IMDwrapPositions <yes/no>
+
+Run your NAMD simulation as usual.
+
+.. seealso::
+   For detailed engine-specific setup instructions, see the `imdclient simulation engine documentation <https://imdclient.readthedocs.io/en/latest/usage.html>`_.
+
+Basic Usage
+===========
+
+Connecting to a Running Simulation
+-----------------------------------
+
+Once your simulation is running with IMD enabled:
+
+.. code-block:: python
+
+   import MDAnalysis as mda
+   
+   # Connect to the simulation
+   u = mda.Universe("topol.tpr", "imd://localhost:8889", buffer_size=10*1024*1024)
+   
+   # Streaming analysis loop
+   for ts in u.trajectory:
+       print(f"Time: {ts.time:.2f} ps, Step: {ts.data.get('step', 'N/A')}")
+       
+       # Your analysis code here
+       selected_atoms = u.select_atoms("protein and name CA")
+       center_of_mass = selected_atoms.center_of_mass()
+       print(f"Protein COM: {center_of_mass}")
+       
+       # Optional: break on some condition
+       if ts.time > 1000:  # Stop after 1000 ps
+           break
+
+Real-time Quality Control
+-------------------------
+
+Monitor simulation health in real-time:
+
+.. code-block:: python
+
+   import MDAnalysis as mda
+   import numpy as np
+   
+   u = mda.Universe("system.tpr", "imd://localhost:8889")
+   
+   previous_positions = None
+   
+   for ts in u.trajectory:
+       current_positions = u.atoms.positions.copy()
+       
+       # Check for simulation artifacts
+       if previous_positions is not None:
+           displacement = np.linalg.norm(current_positions - previous_positions, axis=1)
```

Comment:
This fails on PBC crossing.

Use PBC-aware distance calculations with [`lib.distances.calc_bonds()`](https://docs.mdanalysis.org/stable/documentation_pages/lib/distances.html#MDAnalysis.lib.distances.calc_bonds). This will also demonstrate why it's important that we transfer box information!